Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/utilities.adb

```diff
@@ -0,0 +1,72 @@
+
+with Ada.Exceptions; use Ada.Exceptions;
+with Ada.Text_IO; use Ada.Text_IO;
+with Ada.Strings.Unbounded; use  Ada.Strings.Unbounded;
+
+with gl.Context;
+with GL.Errors;
+with GL.Objects.Shaders.Lists;
+with GL.Types; use  GL.Types;
+
+
+package body Utilities is
+
+      procedure Show_Shader_Program_Data(aProgram : gl.Objects.Programs.Program) is
+        use GL.Objects;
+        Shaders_List     : Shaders.Lists.List := Programs.Attached_Shaders(aProgram);
+        List_Cursor        : Shaders.Lists.Cursor := Shaders_List.First;
+        Shader1             : Shaders.Shader := Shaders.Lists.Element(List_Cursor);
+        Shader_Count   : Positive := 1;
+    begin
+        Put_Line("Shader: " & Positive'Image(Shader_Count));
+        Put_Line(Shaders.Source(Shader1));
+  --      Shader1.Delete_Id;
```

Comment:
I'm not sure what you mean by "We don't need this code anymore".
I've deleted the commented out lines but, otherwise, lines 20 onwards are needed as Show_Shader_Program_Data is called by Initialization.Startup.
I'm wondering if I should delete the exception code which I included for development but is not really needed as part of an example?